uint8[] image
google_chat_ros/OnClick on_click
float64 aspect_ratio # The aspect ratio of this image (width/height). If unset, the server fills it by prefetching the image
